etc/scipipe/build_matrix.yaml

```diff
@@ -127,19 +127,19 @@ shebangtron:
 # use 'stack'
 scipipe_release:
   dockerfile:
-    github_repo: lsst-sqre/docker-tarballs
-    git_ref: master
+    github_repo: lsst-dm/docker-tarballs
+    git_ref: main
     dir: ''
   docker_registry:
     repo: lsstsqre/centos
 newinstall:
   dockerfile:
-    github_repo: lsst-sqre/docker-newinstall
-    git_ref: master
+    github_repo: lsst-dm/docker-newinstall
+    git_ref: main
     dir: ''
   docker_registry:
-    repo: ghcr.io/lsst-sqre/docker-newinstall
-    tag: 9-latest
+    repo: ghcr.io/lsst-dm/docker-newinstall
+    tag: 9-latest-9.0.0
```

Comment:
Would be nice if we could reuse the `splenv_ref` at the top of the file.  Would require a code change, and it would be kind of magic, so it's probably not a good idea.

Reply:
I think it could be a good change in the near future.